parts/k8s/kuberneteswindowssetup.ps1

```diff
@@ -190,6 +251,12 @@ c:\k\kubelet.exe --hostname-override=`$global:AzureHostname --pod-infra-containe
     # more time is needed to pull windows server images
     $KubeletCommandLine += " --image-pull-progress-deadline=20m --cgroups-per-qos=false --enforce-node-allocatable=`"`""
     $KubeletCommandLine += " --volume-plugin-dir=`$global:VolumePluginDir"
+     # Configure kubelet to use CNI plugins if enabled.
+    if ($global:AzureCNIEnabled) {
+        $KubeletCommandLine += $global:AzureCNIKubeletOptions
+    } else {
+        $KubeletCommandLine += " --network-plugin=cni --cni-bin-dir=`$global:CNIPath --cni-conf-dir `$global:CNIPath\config"

```

Comment:
Is azure cni and wincni mutually exclusive? $KubeletCommandLine sets wincni in a few lines above and should be removed?

Reply:
Nice catch. I removed it.